simulation.py

```diff
@@ -0,0 +1,97 @@
+from controller import Controller
+
+
+class Simulation:
+	def __init__(self, controller, targets, timesteps=1, sensitivity=10, failsafe=None):
+		""" Run a simulation on a Controller object
+		
+		targets is a list of tuples each containing length & altitude
+		"""
+		
+		self.controller = controller
+		self.targets = [None, *targets]
```

Comment:
Since we will only need one target per launch, (other kinds of targets would be `max velocity`, `max acceleration`, 'max burn time') I think having multiple targets is overcomplicating the Simulation.